src/cosmoscout/Application.cpp

```diff
@@ -510,10 +508,10 @@ void Application::FrameUpdate() {
       double heightDiff    = polar.z / mGraphicsEngine->pHeightScale.get() - surfaceHeight;
 
       if (!std::isnan(polar.x) && !std::isnan(polar.y) && !std::isnan(heightDiff)) {
-        mGuiManager->getStatusBar()->callJavascript("set_user_position",
+        mGuiManager->getGui()->callJavascript("CosmoScout.call", "timeline", "setUserPosition",
```

Comment:
I find `mGuiManager->getGui()->callJavascript("CosmoScout.timeline.setNorthDirection", angle);` much more readable. Should  we replace all `callJavascript`s?

Reply:
Renamed all calls to `CosmoScout.<api>.<method>`.